butterfly-slack/build.gradle

```diff
@@ -0,0 +1,24 @@
+apply plugin: 'java-library'
+
+dependencies {
+    implementation project(':butterfly-api')
+    implementation project(':butterfly-extensions-api')
```

Comment:
Please remove this dependency. I don't believe it is necessary.

Reply:
removed butterfly-extensions-api but kept butterfly-api for access to TransformationListener